src/Symfony/Component/Validator/Constraints/Image.php

```diff
@@ -203,14 +203,14 @@ public function __construct(
             $filenameCharsetMessage,
         );
 
-        $this->minWidth = $minWidth ?? $this->minWidth;
-        $this->maxWidth = $maxWidth ?? $this->maxWidth;
-        $this->maxHeight = $maxHeight ?? $this->maxHeight;
-        $this->minHeight = $minHeight ?? $this->minHeight;
-        $this->maxRatio = $maxRatio ?? $this->maxRatio;
-        $this->minRatio = $minRatio ?? $this->minRatio;
-        $this->minPixels = $minPixels ?? $this->minPixels;
-        $this->maxPixels = $maxPixels ?? $this->maxPixels;
```

Comment:
we don't really have a deprecation layer for this change, do we?

Reply:
Ah, OK. I was thinking inheritance might set those properties, but that's via $options!